src/components/Settings/SettingsSearchBar.tsx

```diff
@@ -61,7 +60,7 @@ export function SettingsSearchBar() {
           }))
       ),
     ],
-    [settings.state.context]
+    [settings]
```

Comment:
Remember that this will only trigger useEffect if the settings object _reference_ changes! Not sure if that's what we want but wanted to point it out.

Reply:
Yeah I wanted the same behavior as before, but we just no longer need to peel off two layers to get to it.